batch/sql/estimated-current.sql

```diff
@@ -1362,7 +1433,7 @@ BEGIN
     # there are no cancellable jobs left, they have been cancelled
     DELETE FROM batch_inst_coll_cancellable_resources WHERE batch_id = in_batch_id;
 
-    INSERT INTO batches_cancelled VALUES (in_batch_id);
+    INSERT INTO batches_cancelled (id) VALUES (in_batch_id);
```

Comment:
To confirm my understanding, cancelling the batch means a row that is `(batch_id, NULL)` and cancelling a job group means a row that is `(batch_id, job_group_id)`?

Reply:
No. There's an implicit default value here of job_group_id = 0 (`INT DEFAULT 0`). I can explicitly add the job_group_id being 0 here for clarity.